lib/YaST/Warning/Notification.pm

```diff
@@ -0,0 +1,52 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: The class introduces methods in Expert Partitioner to handle
+# a Warning Dialog.
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+package YaST::Warning::Notification;
+use strict;
+use warnings;
+
+sub new {
+    my ($class, $args) = @_;
+    my $self = bless {
+        app => $args->{app}
+    }, $class;
+    return $self->init();
+}
+
+sub init {
+    my $self = shift;
+    $self->{btn_ok}      = $self->{app}->button({id => 'ok_msg'});
+    $self->{lbl_header}  = $self->{app}->label({label => 'Warning'});
+    $self->{lbl_warning} = $self->{app}->label({type  => 'YLabel'});
+    return $self;
+}
+
+sub confirm {
+    my ($self) = @_;
+    YuiRestClient::Wait::wait_until(object => sub {
```

Comment:
makes more sense to have the 'wait_until' in 'is_shown' in case some other method would need it, right?

Reply:
Yeah, maybe we can have `wait_for_shown` method and then use it in `confirm`. I do not want to have wait in `is_shown` method, because we still should have ability to check presence of the element immediately.